corehq/apps/hqadmin/reporting/reports.py

```diff
@@ -294,13 +301,7 @@ def get_active_users_data(domains, datespan, interval, datefield='date',
                 .size(0)
                 .run()
                 .facets.user.result
-                if u['term'] in (
-                    UserES()
-                    .show_inactive()
-                    .mobile_users()
-                    .domain(domains)
-                    .run().doc_ids
-                )
+                if u['term'] in mobile_users
```

Comment:
@nickpell Would using FormES.user_id(mobile_users) accomplish the same thing without using the if statement, or are those referring to different ids?